getting_started/setup_vm/roles/openenclave/vars/common.yml

```diff
@@ -1,5 +1,5 @@
 workspace: "/tmp/"
-oe_ver: "v0.9.0"
+oe_ver: "update_snmalloc_0.5"
```

Comment:
Recent OE master w/ snmalloc rev'ed up to 0.5, which is ideally what we'd like to have as the 0.10 release.